### ISSUE TITLE
Adjust playback speed

### ISSUE THREAD

**Issue by luisherranz** [cogentapps/chat-with-gpt#94]
It would be great to be able to adjust the playback speed 🙂

**Comment by cogentapps** [comment]
Good idea! Unfortunately ElevenLabs doesn't provide a speed option, but there may be a way to implement it in the app anyway.

**Comment by magicmars35** [comment]
Would that be possible with natives Browser Voices (pitch, and speed) ?

**Comment by cogentapps** [comment]
Yes, that's possible to add.

**Comment by eduwass** [comment]
+1, this would be a great addition :+1:
